docs/iot-platform-sdk/running-the-demo.md

```diff
@@ -645,11 +605,14 @@ o.s.i.t.t.To0ServiceApplication.logStarting - Starting To0ServiceApplication
 o.s.b.w.e.tomcat.TomcatWebServer.start - Tomcat started on port(s): 8049 (https) with context path ''
 o.s.i.t.t.To0ServiceApplication.logStarted - Started To0ServiceApplication in 9.159 seconds (JVM running for 10.693)
 ```
+!!! NOTE
+    By default, To0Scheduler is configured to verify Rendezvous's incoming server certificate during TLS handshake for all outgoing HTTPS requests to Rendezvous, made during TO0. To disable this certificate verification for demo purposes, set the application property 'org.sdo.to0.tls.test-mode' (ORG_SDO_TO0_TLS_TEST_MODE in \<sdo-iot-platform-sdk-root>/demo/to0scheduler/to0scheduler.env) to 'false'.
```

Comment:
**Rendezvous service's** incoming server 
HTTPS requests to the **Rendezvous service**.
Can be removed **, made during TO0**